src/test/java/com/google/api/codegen/grpcmetadatagen/PackageMetadataGeneratorTest.java

```diff
@@ -81,15 +81,15 @@ protected Object run() throws Exception {
     options.set(GrpcMetadataGenerator.LANGUAGE, language);
     Map<String, Doc> generatedDocs = new GrpcMetadataGenerator(options).generate(model);
 
-    if (language == "java") {
-      return new ImmutableMap.Builder<String, Doc>().putAll(generatedDocs).build();
-    } else {
+    if (language == "python") {
```

Comment:
we have some language name constants in `MainGapicProviderFactory` - maybe we could make a centralized class with language name constants, and have `MainGapicProviderFactory` reference those language name constants, plus here too.